sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -122,6 +132,27 @@ export interface Contacts {
     readonly id?: string;
 }
 
+// @public
+export class CreateCertificatePoller extends Poller<CreateCertificatePollOperationProperties, CertificateOperation> {
```

Comment:
why is this public?

Reply:
The type should probably be public, so that people can use it. Does it hurt?

Reply:
oh I goofed up here, I thought this was the function to create a certificate poller, not the poller subclass itself. I blame this mistake on being sick 😷